docs/articles/media-types.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Media Types
+parent: Articles
+---
+
+# Media Types
+{:.no_toc}
+
+{: .nb }
+> For detailed walkthroughs of specific TROLIE use cases, please see
+> [Usage Examples](../usage-examples). The article focuses on how the various
+> messages required for those use cases are specified in this standard. If you
+> are unfamiliar with Web APIs in general or HTTP headers in particular, this
+> article is for you.
+
+## Overview
+{:.no_toc}
+
+TROLIE specifies an API that makes use of long-standing features of the Web,
+including [Conditional GET](./conditional-GET) and the subject of this article:
+media types.
+
+> "HTTP uses media types in the Content-Type and Accept  header fields in order
+> to provide open and extensible data typing and type negotiation. **Media types
+> define both a data format and various processing models**: how to process that
+> data in accordance with the message context."
+>
+> Source [RFC 9110 HTTP Semantics](https://www.rfc-editor.org/rfc/rfc9110.html#media.type)
+> (emphasis added)
+
+As an OpenAPI specification, TROLIE uses [JSON
+Schema](https://swagger.io/docs/specification/data-models/keywords/) to describe
+the data formats of [media
+types](https://swagger.io/docs/specification/media-types/), so to define
+processing models for these media types, TROLIE only needs to describe
+the semantics of the transmitted JSON. For the most part, the [TROLIE OpenAPI
+Specification](../spec) does a decent job of explaining what each of the JSON
+elements mean, but that can be unfamiliar to navigate. This article aims to give
```

Comment:
```suggestion
the semantics of the transmitted JSON. While the [TROLIE OpenAPI
Specification](../spec) explains what each of the JSON elements mean, 
it assumes that the reader understands how to use media types in an
API. This article aims to give
```

Just my reaction as I read this; I don't think it read quite how you intended it to read.

Reply:
I believe I addressed this feedback https://github.com/trolie/spec/pull/161/files#diff-e217f8b4ad8a310b77585ef760bf29ae57912123ae78b8e802547d0b3a598540R35